packages/test/test-utils/src/containerUtils.ts

```diff
@@ -89,3 +97,16 @@ export async function getDataStoreEntryPointBackCompat<T>(dataStore: IDataStore)
 	assert(response.status === 200, "empty request should return data object");
 	return response.value as T;
 }
+
+/**
+ * @internal
+ */
+export function assertIsIDeltaManagerFull(
+	deltaManager: IDeltaManager<ISequencedDocumentMessage, IDocumentMessage>,
+): IDeltaManagerFull {
+	assert(
+		"inbound" in deltaManager && "outbound" in deltaManager,
+		"Delta manager does not have inbound/outbound queues.",
+	);
+	return deltaManager as IDeltaManagerFull;
```

Comment:
Some use case could be simplified if this allowed `IDeltaManagerErased | IDeltaManager<ISequencedDocumentMessage, IDocumentMessage>`.
The return line could go with `deltaManager as unknown as IDeltaManagerFull` (or if you wanted to be extra careful `deltaManager as unknown as IDeltaManagerErased | IDeltaManager<ISequencedDocumentMessage, IDocumentMessage> as IDeltaManagerFull`).